playground/ocamlorg-playground.opam

```diff
@@ -48,6 +50,6 @@ dev-repo: "git+https://github.com/ocaml/ocaml.org.git"
 pin-depends: [
   ["esbuild.dev" "git+https://github.com/tmattio/opam-esbuild#7738af5912cc9f554d31da81d2334e88052102fb"]
   ["code-mirror.dev" "git+https://github.com/patricoferris/jsoo-code-mirror#8fe48910e265ff87f9fc94ceb7b3d19fac102a96"]
-  ["merlin-js.dev" "git+https://github.com/tmattio/merlin-js#74edff465942cc4f0b3b51b3f8264db53a460aa2"]
+  ["merlin-js.dev" "git+https://github.com/jonludlam/merlin-js#276d0af663db6aa0bc1bf8733eca206cf91dbcb7"]
```

Comment:
Testing and reviewing is clearly not blocked by this, but should we aim to upstream merlin-js patches (from both my fork and yours) before we merge?

cc @voodoos

Reply:
We should definitely have a longer term plan for this, rather than playing 'pass the repo' :-D

Reply:
Yes, it would be best to upstream the patches, and I can even cut a release :-)